src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -30,83 +38,96 @@ public function __construct($config, $logCallback = null)
      */
     public function createTestUser($name, array $permissions)
     {
+        $app_token = $this->appId . '|' . $this->appSecret;
```

Comment:
wow, i was sure it is there :smiley: nonetheless, we don't use underscore for php entities